tpcx_bb/xbb_tools/cluster_startup.py

```diff
@@ -50,6 +49,7 @@ def get_bsql_config_options():
     config_options['BLAZING_CACHE_DIRECTORY'] = os.environ.get("BLAZING_CACHE_DIRECTORY", '/tmp/')
     config_options['LOGGING_LEVEL'] = os.environ.get("LOGGING_LEVEL", "trace")
     config_options['MAX_JOIN_SCATTER_MEM_OVERHEAD'] = os.environ.get("MAX_JOIN_SCATTER_MEM_OVERHEAD", 500000000)
+    config_options['PROTOCOL'] = os.environ.get("PROTOCOL", "TCP")
```

Comment:
If PROTOCOL is not set, but BlazingContext is passed a dask `client` that is attached to a UCX-enabled cluster, will this still try to default to TCP?

Reply:
Yes

Reply:
Hmm. We could work around this by introducing state into this function, but I think that may be solving the problem at the wrong level.

Do you think it makes sense for the BlazingContext, if attaching to an existing Dask cluster, to choose the PROTOCOL based on the running cluster? cc @williamBlazing for your thoughts as well

Reply:
Yes, make sense to choose the PROTOCOL based on the running cluster. We are missing this feature I think.

Reply:
I'll file an issue